scripts/build.actions.js

```diff
@@ -111,17 +129,16 @@ class BuildActions extends BaseScript {
           }
           return resolve(stats)
         }))
-
-        // zip the bundled file
-        // the path in zip must be renamed to index.js even if buildFilename is not index.js
-        const zipSrcPath = path.join(outBuildDir, outBuildFilename)
-        if (fs.existsSync(zipSrcPath)) {
-          await utils.zip(zipSrcPath, outPath, 'index.js')
-          fs.removeSync(zipSrcPath) // remove the build file
-        } else {
-          throw new Error(`could not find bundled output ${zipSrcPath}, building action '${name}' has likely failed`)
-        }
       }
+
+      // zip the dir
+      await utils.zip(tempBuildDir, outPath)
+      // fs.remove(tempBuildDir) // remove the build file, don't need to wait ...
```

Comment:
should all the commented code here be deleted?

Reply:
I will uncomment, this is a convenience to verify what goes into the zip